src/main/java/ru/skypro/homework/dto/CommentsDto.java

```diff
@@ -0,0 +1,16 @@
+package ru.skypro.homework.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Builder
+public class CommentsDto {
+    private int count;
+    private List<CommentDto> results;
```

Comment:
Тут поправьте вот так, чтобы открывалось объявление
```suggestion
    private List<CommentDto> results = new ArrayList<>();
```